chart/README.md

```diff
@@ -36,7 +36,8 @@ cluster using the [Helm](https://helm.sh) package manager.
 
 ## Features
 
-* Supported executors: ``LocalExecutor``, ``CeleryExecutor``, ``KubernetesExecutor``, ``LocalKubernetesExecutor``, ``CeleryKubernetesExecutor``
+* Supported executors (all Airflow versions): ``LocalExecutor``, ``CeleryExecutor``, ``KubernetesExecutor``
+* Supported executors (Airflow version ``2.0+``): ``LocalKubernetesExecutor``, ``CeleryKubernetesExecutor``
```

Comment:
To a lot of folks 2+ would mean version two and beyond, but in this case it's _just_ version 2

```suggestion
* Supported executors (Airflow version ``2.X.X``): ``LocalKubernetesExecutor``, ``CeleryKubernetesExecutor``
```

or

```suggestion
* Supported executors (Airflow version ``<3.0``): ``LocalKubernetesExecutor``, ``CeleryKubernetesExecutor``
```